imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -183,7 +183,7 @@ Meteor.methods({
     let result;
     try {
       const refundResult = StripeApi.methods.createRefund.call({ refundDetails });
-      Logger.info(refundResult);
+      Logger.debug(refundResult);
```

Comment:
I know this was already like this before changing it to debug, but while we're here, do we want to consider adding more detail to logs like this?  Otherwise, that'll just output some object in the console and you won't be able to tell where it came from or why it was output.  Possibly a good candidate for the child logger API?  (See [how I did that in Launchdock](https://github.com/reactioncommerce/launchdock/blob/development/server/methods/stacks.js#L15-L19))